tinydrive/tinydrive-api/standalone.md

```diff
@@ -83,6 +142,40 @@ tinydrive.start({
 });
 ```
 
+## `tinydrive.controls.start`
+
+Start is similar to pick and browse, but users do not have a way of closing the dialog with this method. This can be useful when launching {{site.cloudfilemanager}} from a URL where there is no application to insert the files into. A [Control Object](#controlobject) is returned for controlling the {{site.cloudfilemanager}} lifecycle, which is not handled by {{site.cloudfilemanager}} when using the `controls` APIs. For a complete list of available settings for this method, see: [Picker settings](#pickersettings).
+
+### Example: Using `tinydrive.controls.start`
+
+```js
+var controls = tinydrive.start({
+  token_provider: '/your-local/jwt-provider'
+}, function() {
+  setTimeout(function() {
+    controls.close();
+  }, 0);
+}, function(error) {
+  console.log(error);
+
+  setTimeout(function() {
+    controls.close();
+  }, 0);
+});
+```
+
+## Control Object
+
+When using callback-based methods, a control object is returned by the function. This object is used to control the {{site.cloudfilemanager}} lifecycle for the view generated by the call. The view must be closed using the control object or it will remain open indefinitively.
+
+As the callbacks may be called before the function has returned, {{site.companyname}} recommends adding calls to this object from the callbacks into `setTimeout` or similar, to prevent calling it before it is spawned.
```

Comment:
This is not acceptable sorry... and goes against what I was lead to believe in the original Drive PR (not sure if it wasn't clear or I misunderstood sorry). If this is needed/recommended then we need to change the implementation so this isn't going to be an issue.